app/src/common/shared/org/mozilla/vrbrowser/ui/BrowserWidget.java

```diff
@@ -122,7 +122,7 @@ public WidgetPlacement getPlacement() {
     @Override
     public void handleTouchEvent(MotionEvent aEvent) {
         if (aEvent.getActionMasked() == MotionEvent.ACTION_DOWN) {
-            Log.e(LOGTAG, "************** REQUESTING FOCUS");
+            Log.d(LOGTAG, "************** REQUESTING FOCUS");
```

Comment:
I would remove this log as it was just there for debugging during development. I'm not sure it adds any value now.